src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs

```diff
@@ -1152,5 +1152,69 @@ public static float GetFontSizeScale(SystemSettingsFontSize systemSettingsFontSi
 
             return ret;
         }
+
+        /// <summary>
+        /// Copy and return the selected text.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static string CopyToClipboard(TextEditor textEditor)
```

Comment:
@SaraMohSamara 

Please add description for each method's parameter and return value.
You can refer to other description for doxygen.

```
        /// <summary>
        /// Converts a @p pixelBuffer with pixel format A8 to RGBA8888 using the given @p color.
        /// @note Does nothing if the @p pixelBuffer is not A8.
        /// </summary>
        /// <param name="pixelBuffer">The pixel buffer with pixel format A8</param>
        /// <param name="color">The color used to convert to RGBA8888</param>
        /// <param name="multiplyByAlpha">multiplyByAlpha Whether to multiply the @p color with the alpha value of the @p pixel @p buffer.</param>
        /// <returns>The pixel buffer converted to RGBA8888.</returns>
        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
        [EditorBrowsable(EditorBrowsableState.Never)]
        public static PixelBuffer ConvertToRgba8888(PixelBuffer pixelBuffer, Vector4 color, bool multiplyByAlpha)
```

Reply:
Done.
Please check and advise.